### ISSUE TITLE
Crash when on debugging view hierarchy.

### ISSUE THREAD

**Issue by larryonoff** [SwipeCellKit/SwipeCellKit#135]
I have the following error when try to debug view hierarchy.

Accessibility isn't configured for my views. Action title is also nil, but image not.

```
Fatal error: You must provide either a title or an image for a SwipeAction: file /Users/larryonoff/Develop/Projects/work/rocketbody/ios/Pods/SwipeCellKit/Source/SwipeTableViewCell+Accessibility.swift, line 88
2018-01-06 13:20:45.801471+0300 RocketBody[7964:2947206] Fatal error: You must provide either a title or an image for a SwipeAction: file /Users/larryonoff/Develop/Projects/work/rocketbody/ios/Pods/SwipeCellKit/Source/SwipeTableViewCell+Accessibility.swift, line 88
Error:    Unable to capture view hierarchy.
Details:  Log Title: Data source expression execution failure.
Log Details: (null)

Log Method: -[DBGDataSourceConnectionLibViewDebugger _executeLLDBExpression:forRequest:onPotentialThread:iteration:]_block_invoke_2
Method:   -[DBGViewDebugger updateDebugHierarchy]_block_invoke_2
Please file a bug at http://bugreport.apple.com with this warning message and any useful information you can provide.
```

**Comment by kurabi** [comment]
The quickest way to fix this is to kindly set the `accessablityLabel` property on the `SwipeAction`. This helps folks with accessibility given that your action does not have a title. 

I'll look at fixing the crash by handling this scenario differently.

**Comment by larryonoff** [comment]
> The quickest way to fix this is to kindly set the `accessablityLabel`

Thank you. This's what I already did.

**Comment by niyaoyao** [comment]
I have the same issue.

**Comment by niyaoyao** [comment]
@larryonoff Maybe you can set a action title when debug, when the scheme run release set title is nil. Code as below.
```
var stickTitle: String? = nil
            #if DEBUG
                stickTitle = "Stick"
            #endif
```
It worked for me.It seems that the SwipeAccessibilityCustomAction initialize will throw a  
`fatalError("You must provide either a title or an image for a SwipeAction")`
,so that Xcode couldn't capture the hierarchy.

**Comment by kurabi** [comment]
Fixed as part of PR #156
